### ISSUE TITLE
Translation improvements

### ISSUE THREAD

**Issue by DenB10** [ssborbis/ContextSearch-web-ext#37]
I'm trying to improve the localization.
So I submit you a link to download a zip file containing several modified files:
http://denb10.free.fr/contextsearch_web_ext-1.4.9.1-modifications.zip

**Comment by ssborbis** [comment]
Looks good. I'll merge with my changes and upload to the repo soon

**Comment by ssborbis** [comment]
Merged and released v1.4.9.2

**Comment by DenB10** [comment]
More improvements here (if you agree):
http://denb10.free.fr/contextsearch_web_ext-1.5.0-modifications.zip

**Comment by ssborbis** [comment]
Added to the repo. I left the help.png large. I can batch resize later but
I'm ok with leaving them a bit higher res

**Comment by ssborbis** [comment]
Whoops, I was checking for true/false and contentEditable was throwing
"inherit".  Should be fixed in 1.5.2 going up now.

**Comment by ssborbis** [comment]
Sorry, wrong contact. Had a few windows open.

**Comment by DenB10** [comment]
OK, no problem!

**Comment by ssborbis** [comment]
Yeah that's my bad for being sloppy with the commits. I switched to using git today and cleaned some things up, including adding help.png to the builds
